test/Microsoft.AspNetCore.Mvc.RazorPages.Test/Internal/PageActionInvokerTest.cs

```diff
@@ -1164,13 +1168,13 @@ public class PageActionInvokerTest : CommonResourceInvokerTest
                 tempDataFactory = Mock.Of<ITempDataDictionaryFactory>(m => m.GetTempData(It.IsAny<HttpContext>()) == Mock.Of<ITempDataDictionary>());
             }
 
-            Func<PageContext, ViewContext, object> pageFactory = (context, viewContext) =>
+            object pageFactory(PageContext context, ViewContext viewContext)
```

Comment:
FYI Yup, VS recommended an inline function here. Might be the first time I've seen that recommendation.

Reply:
Yeah this is much cleaner. The old code was icky.